example_test.go

```diff
@@ -756,6 +756,13 @@ func ExampleScheduler_StartAt() {
 	s.StartBlocking()
 }
 
+func ExampleScheduler_StartAt_afterTaskStart() {
+	s := gocron.NewScheduler(time.UTC)
+	_, _ = s.Every(5).Seconds().Do(task)
+	s.StartAt(time.Now().Add(10 * time.Second))
```

Comment:
Hi @BranislavLazic thanks for taking a stab at this. The solution you have here actually changes / breaks the design of the scheduling chain. The `inScheduleChain` bool is to prevent this type of activity as it causes more pain confusion to users not knowing which job exactly is being changed.

The chain is started anytime a method is called on the scheduler and terminated with `Do` is called.

The actual issue where a divide by zero can happen is when startAt is called and the time is in the past. I think just wrapping the potential divide by zero case in a check if the denominator is zero first will be the way to solve it. I'll open a PR